src/ctapipe/reco/impact.py

```diff
@@ -84,20 +87,25 @@ class ImPACTReconstructor(HillasGeometryReconstructor):
     - Distance units in metres
     - Energy units in TeV
 
+    Parameters
+    ----------
+    subarray : ctapipe.instrument.SubarrayDescription
+        The telescope subarray to use for reconstruction
+    atmosphere_profile : ctapipe.AtmosphereDensityProfile
+        Density vs. altitude profile of the local atmosphere
+    dummy_reconstructor : bool, optional
```

Comment:
I think we should remove this option and find another way. A class under test should not know it's being tested. We should be able to mock the templates in the unit tests without having to have specific code for testing inside the class.

Reply:
Ok, but I would make that a separate PR because it would likely also change the tests